tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -180,3 +181,84 @@
 
 mne.viz.plot_compare_evokeds(evoked_dict, combine="mean", ci=0.95,
                              colors=color_dict, styles=styles_dict)
+
+
+###############################################################################
+# View topographic representation of activity
+# -------------------------------------------
+#
+# Next we view how the topographic activity changes throughout the response.
+
+times = np.arange(-3.5, 13.2, 3.0)
+epochs['Tapping'].average(picks='hbo').plot_joint(times=times)
+
+
+###############################################################################
+# Compare tapping of left and right hands
+# ---------------------------------------
+#
+# Finally we generate topo maps for the left and right conditions to view
+# the location of activity. First we visualise the HbO activity.
+
+times = np.arange(4.0, 11.0, 1.0)
+epochs['Tapping/Left'].average(picks='hbo').plot_topomap(times=times)
+epochs['Tapping/Right'].average(picks='hbo').plot_topomap(times=times)
+
+###############################################################################
+# And we also view the HbR activity for the two conditions.
+
+epochs['Tapping/Left'].average(picks='hbr').plot_topomap(times=times)
+epochs['Tapping/Right'].average(picks='hbr').plot_topomap(times=times)
+
+###############################################################################
+# And we can plot the comparison at a single time point for two conditions.
+
+fig, axes = plt.subplots(nrows=2, ncols=4, figsize=(9, 5))
+vmin, vmax, ts = -8, 8, 9.0
+epochs['Tapping/Left'].average(picks='hbo').plot_topomap(ch_type='hbo',
```

Comment:
everytime you do `epochs["Tapping/Left"].average(picks='hbo')` you make a copy of epochs then average. Can you factorize this computation. I would get `evoked_left` and `evoked_right`.

Reply:
Thanks, that made me find that I should have had `fnirs=ch_type` not `fnirs=true` in pick_types. Should be fixed now.